webhook/client/client.go

```diff
@@ -23,7 +23,7 @@ type Client struct {
 	Datastore *datastore.DataStore
 }
 
-func New(ctx context.Context, config *rest.Config, namespace string) (*Client, error) {
+func New(ctx context.Context, stopCh chan struct{}, config *rest.Config, namespace string) (*Client, error) {
```

Comment:
NIT: If you use the cancellable context you also don't need to pass the extra stop channel